cfme/infrastructure/virtual_machines.py

```diff
@@ -889,3 +899,30 @@ def resetter(self, *args, **kwargs):
 
     def am_i_here(self, *args, **kwargs):
         return match_page(summary='Template "{}"'.format(self.obj.name))
+
+
+@navigator.register(Vm, 'ProvisionVM')
+class ProvisionVM(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self, *args, **kwargs):
+        lcl_btn("Provision VMs")
+
+        # choosing template and going further
+        template_select_form.template_table._update_cache()
+        template = template_select_form.template_table.find_row_by_cells({
+            'Name': self.obj.template_name,
+            'Provider': self.obj.provider.name
+        })
+        if template:
+            sel.click(template)
+            # In order to mitigate the sometimes very long spinner timeout, raise the timeout
+            with sel.ajax_timeout(90):
+                sel.click(form_buttons.FormButton("Continue", force_click=True))
+
+        else:
+            raise TemplateNotFound('Unable to find template "{}" for provider "{}"'.format(
+                self.obj.template_name, self.obj.provider.key))
+
+    def am_i_here(self, *args, **kwargs):
```

Comment:
I'm not sure we can have this here. There is no specificity, we could be on the Provision page for another Vm, I think that resetting here and having no am_i_here is fine.